pkg/services/accesscontrol/ossaccesscontrol/ossaccesscontrol.go

```diff
@@ -82,28 +84,49 @@ func (ac *OSSAccessControlService) GetUserPermissions(ctx context.Context, user
 	timer := prometheus.NewTimer(metrics.MAccessPermissionsSummary)
 	defer timer.ObserveDuration()
 
-	builtinRoles := ac.GetUserBuiltInRoles(user)
+	permissions := ac.getFixedPermissions(ctx, user)
```

Comment:
Nit. you could provide the user's built-in role to call `ac.GetUserBuiltInRoles` once. (however that's not an expensive call)